src/UnigramWordPredictor.java

```diff
@@ -101,7 +120,18 @@ public void train(Scanner scanner) {
   public String predictNextWord(List<String> context) {
     // TODO: Return a predicted word given the words preceding it
     // Hint: only the last word in context should be looked at
-    return null;
+
+    // create random class instance
+    Random r = new Random();
+
+    // get the last word from the list
+    String lastWord = context.get(context.size() - 1);
+
+    // store the words that follow lastWord in List variable
+    List<String> followupWords = neighborMap.get(lastWord);
+
+    // return the word that was randomly selected using Random class
+    return followupWords.get(r.nextInt(followupWords.size()));
```

Comment:
This works great! But what would happen if we haven't seen the word in the context before or if it was the last word in the file and duplicated nowhere else? What would your code currently do? What options would you have to deal with this?